extensions/twenty/src/services/TwentySDK.ts

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { getPreferenceValues } from "@raycast/api";
+import { Api } from "../enum/api";
+import fetch from "node-fetch";
+import { getActiveDataModelsSchema } from "./zod/schema/dataModelSchema";
+import { dataModelsFieldsSchema } from "./zod/schema/recordFieldSchema";
+
+class TwentySDK {
+  private url!: string;
+  private token!: string;
+
+  constructor() {
+    const { token, url }: { token: string; url: string } = getPreferenceValues();
+    this.token = `Bearer ${token}`;
+    this.url = url ? `${url}/rest` : `https://api.twenty.com/rest`;
+  }
+
+  async getActiveDataModels() {
+    const response = await fetch(this.url + "/metadata/objects", {
+      method: "GET",
+      headers: {
+        "Content-Type": "application/json",
+        [Api.KEY]: this.token,
+      },
+    });
+    const rawData = await response.json();
+    const data = getActiveDataModelsSchema.parse(rawData);
+    const activeDataModel = data.data.objects.filter((model) => !model.isSystem && model.isActive);
+    return activeDataModel;
+  }
+
+  async getRecordFieldsForDataModel(id: string) {
+    try {
+      const response = await fetch(this.url + "/metadata/objects", {
+        method: "GET",
+        headers: {
+          "Content-Type": "application/json",
+          [Api.KEY]: this.token,
+        },
+      });
+      const data = (await response.json()) as any;
+      const objectRecords = await dataModelsFieldsSchema.parseAsync(data?.data?.objects);
+      const objectRecordWithFieldsMetadata = await objectRecords.find((object) => object.id === id)!;
```

Comment:
`await` is redundant.
```suggestion
      const objectRecordWithFieldsMetadata = objectRecords.find((object) => object.id === id)!;
```